rts/System/Config/ConfigVariable.h

```diff
@@ -142,13 +146,25 @@ class ConfigVariableBuilder : public spring::noncopyable
 	ConfigVariableBuilder(ConfigVariableTypedMetaData<T>& data) : data(&data) {}
 	const ConfigVariableMetaData* GetData() const { return data; }
 
+	ConfigVariableBuilder& declarationFile(const char* file) {
+		std::string_view svFile(file);
+		auto off = std::min(svFile.rfind("/rts/"), svFile.rfind("\\rts\\"));
+		if unlikely(off == std::string::npos) {
+			data->declarationFile = std::string(file);
+			return *this;
+		}
+
+		svFile.remove_prefix(off);
+		data->declarationFile = std::format("{}{}", ".", svFile);
```

Comment:
As discussed here, could remove the dot and the leading slash:

```suggestion
		svFile.remove_prefix(off + 1);
		data->declarationFile = std::string(svFile);
```